package.json

```diff
@@ -18,7 +18,7 @@
     "coverage": "istanbul report text && istanbul report html",
     "test:fast": "TEST=1 mocha --opts test/fast-mocha.opts",
     "test:debug": "npm run test:build && TEST=1 mocha --inspect-brk",
-    "prettify": "balena-lint -e ts -e js --typescript --fix src/ test/ typings/ webpack.config.js",
+    "prettify": "balena-lint -e ts -e js --typescript --fix src/ test/ typings/",
```

Comment:
This removes webpack.config.js from prettifying because it needs to be able to run directly on node 6 and prettifying it breaks that